platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/NativeMapView.java

```diff
@@ -42,6 +45,8 @@
     // Used for callbacks
     private MapView mMapView;
 
+    private final float pixelRatio;
+
```

Comment:
maybe we should move all density related calls to NativeMapView in favor of MapView